src/silx/conftest.py

```diff
@@ -55,11 +55,11 @@ def pytest_addoption(parser):
         help="Disable the test of the OpenCL part",
     )
     parser.addoption(
-        "--low-mem",
-        dest="low_mem",
-        default=False,
-        action="store_true",
-        help="Disable test with large memory consumption (>100Mbyte",
+        "--no-high-mem",
+        dest="high_mem",
+        default=True,
+        action="store_false",
+        help="Disable tests with large memory consumption (>100Mbytes)",
```

Comment:
I may miss something, but shouldn't this be revert and have a `--high-mem` options that `store_true` instead of a `False` default?